sys/net/application_layer/dhcpv6/client.c

```diff
@@ -707,6 +719,15 @@ static void _solicit_servers(event_t *event)
     msg_len += _compose_elapsed_time_opt(time);
     msg_len += _compose_oro_opt((dhcpv6_opt_oro_t *)&send_buf[msg_len], oro_opts,
                                 ARRAY_SIZE(oro_opts));
+
+    #ifdef CONFIG_DHCPV6_CLIENT_MUD_URL
```

Comment:
```suggestion
    if (IS_USED(MODULE_GNRC_DHCPV6_CLIENT_MUD_URL)) {
```

The config define is now always set

Reply:
This suggestion has been incorporated in the latest commits